src/aus.h

```diff
@@ -0,0 +1,329 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "types.h"
+
+namespace librealsense
+{
+    
+    class aus_value
```

Comment:
Not sure I understand why we have an abstract class if 2 inherent from it and each use different functions..